internal/iam/command_rbac_role_binding.go

```diff
@@ -166,13 +164,7 @@ func addClusterFlags(cmd *cobra.Command, cfg *config.Config, cliCommand *pcmd.CL
 	if cfg.IsCloudLogin() {
 		cmd.Flags().String("environment", "", "Environment ID for scope of role-binding operation.")
 		cmd.Flags().Bool("current-environment", false, "Use current environment ID for scope.")
-
-		dc := dynamicconfig.New(cfg, nil)
-		_ = dc.ParseFlagsIntoConfig(cmd)
-		if cfg.IsTest || featureflags.Manager.BoolVariation("cli.flink.open_preview", dc.Context(), config.CliLaunchDarklyClient, true, false) {
-			cmd.Flags().String("flink-region", "", "Flink region ID for the role binding.")
-		}
-
+		cmd.Flags().String("flink-region", "", "Flink region ID for the role binding.")
```

Comment:
I think this belongs under `--cloud-cluster`, since it's hierarchically lower.

Reply:
they are on the same level, but happy to move it down